app/models/file.js

```diff
@@ -95,15 +95,22 @@ module.exports = Backbone.Model.extend({
       previous: resp
     };
 
+    var defaults = this.get('defaults');
     res.content = resp.replace(/^(---\n)((.|\n)*?)---\n?/, function(match, dashes, frontmatter) {
-      var regex = /published: false/;
 
       try {
         // TODO: _.defaults for each key
         res.metadata = jsyaml.safeLoad(frontmatter);
-
-        // Default to published unless explicitly set to false
-        res.metadata.published = !regex.test(frontmatter);
+        
+        // Default to published unless explicitly set in metadata or defaults.
+        var defaultPublished = _(defaults).find(function(def){
+          return def.name === 'published';
+        });
```

Comment:
does `find({ name: 'published' })` work? a little less overhead because you don't have to create the func there